share/spice/google_plus/google_plus.css

```diff
@@ -1,24 +1,12 @@
-.tile--googleplus {
-    width: 12.4em;
-    height: 17.2em;
+.tile--google_plus .tile__media {
+    height: 10em;
 }
 
-.tile--googleplus .tile__media {
-    height: 12em;
-}
-
-.tile--googleplus  .tile__media__img {
-    margin-top: 1.6em;
+.tile--google_plus  .tile__media__img {
+    margin-top: 1.4em;
     border-radius: 100%;
 }
 
-.tile--googleplus .tile__body {
-    padding: .3em 1em;
-}
-
-.tile--googleplus  .tile__title {
+.tile--google_plus  .tile__title {
     text-align: center;
-    font-size: 1.4em;
-    font-weight: 400;
```

Comment:
I think we should keep the font-weight. We can address the size using the `elClass` property...